### ISSUE TITLE
Add/remove fee by Rest API

### ISSUE THREAD

**Issue by amin3d** [woocommerce/woocommerce#11940]
In the name of Allah,I'm looking for a way to add extra fee to order through the API,I was googling and didn't find anything. Is there any way to remove a fee?

**Comment by amin3d** [comment]
In the name of Allah, hopefully i found a way to do this;

`
$order = $request->get_param( 'order' );
$order = wc_get_order($order);	
$order_fee            = new stdClass();
$order_fee->id        = sanitize_title( 'withdrawal' );
$order_fee->name      = 'withdrawal';
$order_fee->amount    = 1000;
$order_fee->taxable   = false;
$order_fee->tax       = 0;
$order_fee->tax_data  = array();
$order_fee->tax_class = '';
$fee_id = $order->add_fee( $order_fee );
`

**Comment by amjad0** [comment]
I asked the same question here: #11869 

Perhaps an example would be nice. Sending a null did not work for me. Need to know to delete a specific fee? (not all)

**Comment by amin3d** [comment]
In the name of Allah, My problem with adding was solved but to remove or even update the fee I need the id of fee,I know the name of the feed but how do i get id of it?

**Comment by amin3d** [comment]
In the name of Allah, I'v found a way to delete, first i wrote a function to find the id of fee,
```
	private function get_feed_id($order,$name)
	{
		$fees = $order->get_fees();
		foreach($fees as $key=>$fee)
		{
			if($fee['name']==$name)
				return $key;
		}
		
	}
```
and after that i used wc_delete_order_item to remove that fee

`wc_delete_order_item($this->get_feed_id($order,'point_reduction'));`

**Comment by claudiosmweb** [comment]
Closed in favor of #12013
